db/seeds.rb

```diff
@@ -12,6 +12,25 @@
              password_confirmation: 'password',
              admin: false
 
+Field.create! name: 'Curb cut',
+              category: 'Accessible',
+              field_type: :choice,
+              #choices: ["Within 20'", 'No curb cut', 'No curb'],
```

Comment:
I didn't know how to include this in fields. :/

Reply:
Text, and then serialize to an Array.